scripts/yapf_formatter.py

```diff
@@ -0,0 +1,31 @@
+from yapf.yapflib.yapf_api import FormatCode
+from github import Github
+import sys
+import re
+import base64
+
+ACCESS_TOKEN = []
+REPO_NAME = []
+WORKING_PATH = []
+ACCESS_TOKEN = sys.argv[1]
+REPO_NAME = sys.argv[2]
+WORKING_PATH = sys.argv[3]
+
+g = Github(ACCESS_TOKEN)
+repo = g.get_repo(REPO_NAME)
+contents = repo.get_contents(WORKING_PATH)
+
+pyscripts = []
+for content_file in contents:
+    result = re.findall(r'\.[^.\\/:*?"<>|\r\n]+$', content_file.name)
+    if result[0] == '.py':
+        pyscripts.append(content_file)
+
+for pyscript in pyscripts:
+    py_content = repo.get_contents(pyscript.path).content
+    py_content = base64.b64decode(py_content).decode('utf-8')
+    py_content = FormatCode(py_content)
+    if py_content[1]:
+        message = ("style(" + pyscript.name + "): format "
+                "python script with yapf")
```

Comment:
Do you miss the `+`?

Are the outer brackets necessary?

Reply:
Do you sure yapf only false when nothing to be formatted?

If there are other situations may throw out a `false`, I think it is better to catch the message why it is caused.

Reply:
> Do you miss the `+`?
> 
> Are the outer brackets necessary?

The outer brackets are used to split a whole sentence into two lines and `+` shouldn't be here.

Reply:
> Do you sure yapf only false when nothing to be formatted?
> 
> If there are other situations may throw out a `false`, I think it is better to catch the message why it is caused.

I didn't find any explanation of this return value in document or any other web. But I did some tests showing this return value is only related to the scripts' being formatted or not.